lib/mongoid/criteria/queryable/selector.rb

```diff
@@ -150,6 +150,8 @@ def evolve_multi(specs)
         #
         # @return [ Object ] The serialized object.
         def evolve(serializer, value)
+          return value.raw_value if value.is_a?(Mongoid::RawValue)
```

Comment:
Hey @johnnyshields -- I hate to keep drawing this PR out, but I wanted to ask if there was a specific reason for using a guard here, rather than adding `when Mongoid::RawValue` as a clause in the `case` statement below.

Reply:
Changed it. Good catch.